spec/integration/monorepo/before/Frameworks/CustomHeaderSearchPaths/Sources/PublicClass.m

```diff
@@ -1,5 +1,6 @@
 #import <CustomHeaderSearchPaths/PublicHeader.h>
 #import "Internal/InternalHeader.h"
+#import <libxml/parser.h>
```

Comment:
Worth a comment making it explicit that this is testing that `SDKROOT` needs to be getting set appropriately for SDK supplied headers to work?